components/common/Nav.tsx

```diff
@@ -129,6 +130,13 @@ export default function Nav({ ...props }: FlexProps & PositionProps) {
 
   const toggle = (): void => setIsOpen(!isOpen)
 
+  const closePanel = useCallback(() => setIsOpen(false), [])
+
```

Comment:
I'm not sure using `useCallback()` has any benefits in this context. Is there a specific reason for this change @mwickett?

Reply:
Ah, yeah - because it's a dependency of the `useEffect` here, and the `closePanel` function gets recreated on every render, it causes an unnecessary re-render. `useCallback` prevents this. But, talking this through makes me realize that I can just move `closePanel` into the `useEffect` block!

Reply:
Got it, thanks for the explanation. Looks better now!